### PR TITLE
(CDAP-12307) Supports HA for TMS service

### PR DESCRIPTION
**Pull request by chtyim**

### CODE REVIEW

**Comment by chtyim** [comment]
Build https://builds.cask.co/browse/CDAP-RUT1285

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -1673,7 +1672,7 @@
 
   <property>
     <name>messaging.max.instances</name>
-    <value>1</value>
+    <value>${master.service.max.instances}</value>
     <description>
       Maximum number of instances for the messaging service
```
Comment (sreevatsanraman) [inline cdap-common/src/main/resources/cdap-default.xml]: We should update the description, saying it does not increase scalability but only used for HA.

Comment (chtyim) [reply cdap-common/src/main/resources/cdap-default.xml]: Probably should do the same for tx?

Comment (sreevatsanraman) [reply cdap-common/src/main/resources/cdap-default.xml]: +1

**Comment by sreevatsanraman** [comment]
Minor doc comment. Fix, update md5 and merge. LGTM

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (chtyim) [reply cdap-common/src/main/resources/cdap-default.xml]: Updated.

**Comment by sreevatsanraman** [comment]
LGTM

**Comment by chtyim** [comment]
Test was passing before the last `cdap-default.xml` description update, which won't affect the build. Merging now.

**Pull request merged**